app/components/ProjectVersionLink/index.jsx

```diff
@@ -0,0 +1,37 @@
+/* global React */
+
+/**
+ * Link to open the version page.
+ * Label is project + version name
+ */
+let ProjectVersionLink = React.createClass({
+
+  propTypes: {
+    projectName: React.PropTypes.string,
+    versionSlug: React.PropTypes.string,
+    versionPageUrl: React.PropTypes.string
+  },
+
+  getDefaultProps: () => {
+    return {
+      projectName: 'Loading ',
+      versionSlug: 'Loading '
```

Comment:
I think it would be nicer if this was `Loading…`. It makes it look more animated, even when it isn't.